ion-schema/src/model/constraints/timestamp_precision.rs

```diff
@@ -185,7 +185,7 @@ impl ValidateInternal for TimestampPrecision {
             return recorder.accept(ViolationInfo::new(
                 self.into(),
                 value.clone(),
-                format!("expected a timestamp precision of {range}; found {timestamp}",),
+                format!("expected a timestamp precision of {range:?}; found {timestamp:?}",),
```

Comment:
🗺️ Changed to use the `Debug` representation instead of `Display` as part of dropping `Display` from the trait bounds of `IonSchemaRange`.